Formula/imagemagick.rb

```diff
@@ -72,49 +73,149 @@ def install
       --enable-static
     ]
 
+    if build.without? "freetype"
+      args << "--without-freetype"
+    else
+      args << "--with-freetype"
+    end
+
+    if build.without? "jpeg"
+      args << "--without-jpeg"
+    else
+      args << "--with-jpeg"
+    end
+
+    if build.without? "libpng"
+      args << "--without-png"
+    else
+      args << "--with-png"
+    end
+
+    if build.without? "libtiff"
+      args << "--without-tiff"
+    else
+      args << "--with-tiff"
+    end
+
+    if build.without? "magick-plus-plus"
+      args << "--without-magick-plus-plus"
+    else
+      args << "--with-magick-plus-plus"
+    end
+
     if build.without? "modules"
       args << "--without-modules"
     else
       args << "--with-modules"
     end
 
+    if build.without? "threads"
+      args << "--without-threads"
+    else
+      args << "--with-threads"
+    end
+
+    if build.with? "fftw"
+      args << "--with-fftw"
+    else
+      args << "--without-fftw"
+    end
+
+    if build.with? "fontconfig"
+      args << "--with-fontconfig"
+    else
+      args << "--without-fontconfig"
+    end
+
+    if build.with? "ghostscript"
+      args << "--with-gslib" << "--with-gs-font-dir=#{HOMEBREW_PREFIX}/share/ghostscript/fonts"
```

Comment:
Suspect you'll want to split this onto two unique lines to avoid plowing past 80 characters.